packages/docs/docs/getting-started/code-block.mdx

```diff
@@ -0,0 +1,249 @@
+---
+sidebar_position: 12
+slug: /code-block
+---
+
+import AnimationPlayer from '@site/src/components/AnimationPlayer';
+import signalSource from '!!raw-loader!@motion-canvas/examples/src/scenes/node-signal';
+
+# CodeBlock
+
+<AnimationPlayer banner name="code-block" />
+
+The `CodeBlock` component is used to present syntax highlighted code. It can
+automatically highlight many common languages, and, once displayed, that code
```

Comment:
nit: "It can automatically highlight many common languages. Once displayed, the code can be modified in-place to present changes to the viewer"

Sometimes extra commas make the sentence harder to read, I'd propose we use this sentence structure instead.